tools/lint

```diff
@@ -81,12 +84,14 @@ def run():
 
     linter_config = LinterConfig(by_lang)
     linter_config.external_linter('add_class', ['tools/find-add-class'], ['js'])
-    linter_config.external_linter('css', ['node', 'node_modules/.bin/stylelint'], ['css', 'scss'])
+    linter_config.external_linter('css', ['node', 'node_modules/.bin/stylelint'],
+                                  ['css', 'scss'], args.fix)
     linter_config.external_linter('eslint', ['node', 'node_modules/.bin/eslint',
-                                             '--quiet', '--cache', '--ext', '.js,.ts'], ['js', 'ts'])
+                                             '--quiet', '--cache', '--ext', '.js,.ts'],
+                                  ['js', 'ts'], args.fix)
```

Comment:
I think we might want to make this a bit more flexible, e.g. `fix_arg="--fix"`, since it's possible some third-party tool(s) will use a different name for this feature.